@xen-orchestra/backups/_runners/_writers/IncrementalRemoteWriter.mjs

```diff
@@ -31,7 +31,7 @@ export class IncrementalRemoteWriter extends MixinRemoteWriter(AbstractIncrement
 
     await asyncMap(baseUuidToSrcVdi, async ([baseUuid, srcVdi]) => {
       let parentDestPath
-      const vhdDir = `${vdisDir}/${srcVdi.uuid}`
+      const vhdDir = `/${vdisDir}/${srcVdi.uuid}`
```

Comment:
Do what you want, this commit is unnecessary, this is a bug in `vhd-lib/chainVhd` which I will fix.